cpp/oneapi/dal/algo/jaccard/vertex_similarity_types.hpp

```diff
@@ -67,22 +67,21 @@ class ONEAPI_DAL_EXPORT vertex_similarity_result {
     /// the table of the corresponding computed Jaccard similarity coefficients, and
     /// the number of non-zero Jaccard similarity coefficients in the block.
     ///
-    /// @param [in]   vertex_pairs  The table of size 2*nonzero_coeff_count with
-    ///                             vertex pairs which have non-zero Jaccard
-    ///                             similarity coefficients
-    /// @param [in]   coeffs        The table of size 1*nonzero_coeff_count with
-    ///                             non-zero Jaccard similarity coefficients
-    ///
-    /// @param [in] nonzero_coeff_count The number of non-zero Jaccard coefficients
+    /// @param [in]   vertex_pairs        The table of size [nonzero_coeff_count x 2] with
+    ///                                   vertex pairs which have non-zero Jaccard
+    ///                                   similarity coefficients
+    /// @param [in]   coeffs              The table of size [nonzero_coeff_count x 1] with
+    ///                                   non-zero Jaccard similarity coefficients
+    /// @param [in]   nonzero_coeff_count The number of non-zero Jaccard coefficients
     vertex_similarity_result(const table& vertex_pairs,
                              const table& coeffs,
                              std::int64_t nonzero_coeff_count);
 
-    /// Returns the table of size 1*nonzero_coeff_count with non-zero Jaccard
+    /// Returns the table of size [nonzero_coeff_count x 1] with non-zero Jaccard
```

Comment:
The shape of the table does not depend on the layout - it has `n` rows and `m` columns in both row major and column major formats. The only difference is that in rowmajor format adjacent elements correspond to the same row, and in colmajor - to the same column.
That's the right shape of that table?

Reply:
Yes, it is right